aviatrix/data_source_aviatrix_network_domains.go

```diff
@@ -0,0 +1,120 @@
+package aviatrix
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v2/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAviatrixNetworkDomains() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAviatrixNetworkDomainsRead,
+
+		Schema: map[string]*schema.Schema{
+			"network_domain_list": {
```

Comment:
Since we renamed the resource, we should probably rename this attribute too. Maybe just call it `network_domains` as well. What do you think @shuoliugit @zhiqiangjin?

Reply:
yes, i have fixed as the "network_domains"